designer/server/src/test/scala/pl/touk/nussknacker/ui/api/RemoteEnvironmentResourcesSpec.scala

```diff
@@ -217,6 +213,16 @@ class RemoteEnvironmentResourcesSpec
       Future.successful(Right(testMigrationResults))
     }
 
+    override def migrate(
+        processingMode: ProcessingMode,
+        engineSetupName: EngineSetupName,
+        scenarioWithDetailsForMigrations: ScenarioWithDetailsForMigrations
+    )(implicit ec: ExecutionContext, loggedUser: LoggedUser): Future[Either[NuDesignerError, Unit]] = {
+      val localScenarioGraph = scenarioWithDetailsForMigrations.scenarioGraphUnsafe
```

Comment:
because it's unsafe, maybe we should be sure that the expectation won't be thrown outside the Future?